src/dialogs/config-flow/step-flow-form.ts

```diff
@@ -81,6 +81,23 @@ class StepFlowForm extends LitElement {
     `;
   }
 
+  private _enrichFields(schema: HaFormSchema[]): HaFormSchema[] {
+    const fields = autocompleteLoginFields(schema);
+
+    fields.forEach((field) => {
+      if (
+        "selector" in field &&
+        "select" in field.selector &&
+        field.selector.select &&
+        !field.selector.select.translation_domain
+      ) {
+        field.selector.select.translation_domain = this.step.handler;
```

Comment:
`this.step.handler` will only be the translation domain for the config-flow.
For the option flow this will be the config entry id.